Civi/RemoteTools/JsonSchema/JsonSchema.php

```diff
@@ -241,6 +241,26 @@ public function getKeywordValueAtOrDefault($path, $default) {
     return $keywordValue;
   }
 
+  /**
+   * @return static
+   */
+  public function clone(): self {
+    /** @var static $clone */
+    $clone = (new \ReflectionClass(get_class($this)))->newInstanceWithoutConstructor();
+    $clone->keywords = \array_map(function ($value) {
+      if ($value instanceof self) {
+        return $value->clone();
+      }
+      elseif (\is_array($value)) {
+        return \array_values(\array_map(fn ($value) => $value instanceof self ? $value->clone() : $value, $value));
+      }
+
+      return $value;
+    }, $this->keywords);
+
+    return $clone;
+  }
```

Comment:
Why does this not implement `__clone()`?

Reply:
You're right, using `__clone()` here makes sense. Changed that.